optimization/optimizing-your-react-node-project.md

```diff
@@ -8,7 +8,7 @@ About 1.5 hours
 
 Here are links to lessons that should be completed before this lesson:
 
-- [Intro to Optimization](/optimization/optimization.md)
```

Comment:
@vegetabill also looking for your opinion on this one... is this still a prereq?

Reply:
Any "optimizing/optimization" lesson is elective, so both this and its prereq

Reply:
@frozenGirl-spd please rename this file, and move this link to electives.

Reply:
sorry, I meant keep this and change it to the new link, and make sure it is moved to the electives section in the Readme.

Reply:
move or copy it to elective section in readme?

Reply:
copy

Reply:
please also bring optimization/optimizing-your-react-node-project.md into electives, fix any links to the new path, and put it in the right place in the readme.